doc/source/installation/apt_repo.rst

```diff
@@ -167,11 +167,11 @@ Now, you can install |Percona Server| using |dpkg|. |tip.run-this.root|
 
 This will install all the packages from the bundle. Another option is to
 download/specify only the packages you need for running |Percona Server|
-installation (``libperconaserverclient20_8.0.3-13.stretch_amd64.deb``,
-``percona-server-client-8.0.3-13.stretch_amd64.deb``,
-``percona-server-common-8.0.3-13.stretch_amd64.deb``, and
-``percona-server-server-8.0.3-13.stretch_amd64.deb``. Optionally, you can install
-``percona-server-tokudb-8.0.3-13.stretch_amd64.deb`` if you want the |TokuDB|
+installation (``libperconaserverclient21_8.0.13-3-1.stretch_amd64.deb``,
+``percona-server-client-8.0.13-3-1.stretch_amd64.deb``,
```

Comment:
correct is: `percona-server-client_8.0.13-3-1.stretch_amd64.deb`, same for others below